common/install_magma.sh

```diff
@@ -14,6 +14,9 @@ function do_install() {
     elif [[ ${cuda_version_nodot} == 117 ]]; then
 	MAGMA_VERSION="2.6.1"
 	magma_archive="magma-cuda${cuda_version_nodot}-${MAGMA_VERSION}-0.tar.bz2"
+    elif [[ ${cuda_version_nodot} == 118 ]]; then
```

Comment:
@ptrblck Please remove this if statement  and replace by:

```
MAGMA_VERSION="2.6.1"
magma_archive="magma-cuda${cuda_version_nodot}-${MAGMA_VERSION}-1.tar.bz2"
```

Please refer to: https://anaconda.org/pytorch/magma-cuda116/files
 build_number 1 is default now